src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -355,8 +357,19 @@
   background-clip: padding-box;
   box-shadow: var(--pf-c-dropdown__menu--BoxShadow);
 
-  &.pf-m-align-right {
-    right: 0;
+  @each $breakpoint, $breakpoint-value in $pf-c-dropdown--breakpoint-map {
+    $breakpoint-name: if($breakpoint != "base", -on-#{$breakpoint}, "");
+
+    @include pf-apply-breakpoint($breakpoint, $pf-c-dropdown--breakpoint-map) {
+      &.pf-m-align-right#{$breakpoint-name} {
+        right: 0;
+      }
+
+      // unset alignment set for .pf-m-align-right
+      &.pf-m-align-left#{$breakpoint-name} {
+        right: unset;
```

Comment:
I think I'd just set it to `auto` to avoid any unintended outcome.

Reply:
Will do, thank you!